lib/init.c

```diff
@@ -279,9 +279,11 @@ struct smb2_context *smb2_init_context(void)
         char buf[1024] _U_;
         int i, ret;
         static int ctr;
-
+#ifdef _XBOX
+        srandom(time(NULL) ^ 1 ^ ctr++);
```

Comment:
Can probably get rid of ^1 as it is a constant change and does not improve randomness.
I.e.
srandom(time(NULL) ^ ctr++)

Reply:
Solved using the original code